File: Models/EventConcert.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using Microsoft.AspNetCore.Http;

namespace fortress.core.Model
{
    public class EventConcert
    {
        public int EventConcertId { get; set; }
        
        public string EventName { get; set; }

        public string Flyer { get; set; }

        public DateTime EventDate { get; set; }

        public bool IsApproved { get; set; }
        
        public string UserNotes { get; set; }

        //public string VenueName { get; set; }

        public Venue Venue { get; set; }

        public string Tickets { get; set; }

        public List<Comment> Comments { get; set; }
    }
}

File: ClientApp/src/interface/eventconcert.ts
import { Venue } from "./venue";

export interface EventConcert {
    eventConcertId: number;
    eventName: string;
    flyer: string;
    eventDate: string;
    isApproved: boolean;
    venue: Venue;
    tickets: string;
    eventTime: string;
    details: string;
    price: string;
    dateFormatted:string;
    status: string;
}
